### ISSUE TITLE
Omit type attributes for style sheets and scripts

### ISSUE THREAD

**Issue by jonasspott** [django-webpack/django-webpack-loader#152]
Info from https://passmarked.com/

```
<!-- Not recommended -->
<link rel="stylesheet" href="//www.google.com/css/maia.css" type="text/css">
<!-- Recommended -->
<link rel="stylesheet" href="//www.google.com/css/maia.css">
```

Info:
https://google.github.io/styleguide/htmlcssguide.xml

**Comment by owais** [comment]
Sounds good. Feel free to send a PR to fix it. Thanks.

**Comment by fjsj** [comment]
Fixed by #170
